tests/testHelicalJoint.cpp

```diff
@@ -94,6 +98,26 @@ TEST(Joint, params_constructor) {
                       j1->parameters().scalar_limits.value_limit_threshold));
 }
 
+BOOST_CLASS_EXPORT(gtdynamics::HelicalJoint)
+BOOST_CLASS_EXPORT(gtdynamics::RevoluteJoint)
```

Comment:
is revolute joint needed here?

Reply:
Huh guess not. I don't know why I put it there, guess due to just trying to get the tests to pass.

Reply:
no worries